packages/@aws-cdk/aws-ecs/test/ec2/integ.capacity-provider.ts

```diff
@@ -4,38 +4,48 @@ import * as ecs from '../../lib';
 
 const app = new cdk.App();
 
-const stack = new cdk.Stack(app, 'integ-capacity-provider');
+const env = {
+  region: process.env.CDK_DEFAULT_REGION,
+  account: process.env.CDK_DEFAULT_ACCOUNT,
+};
 
-const vpc = new ec2.Vpc(stack, 'Vpc', { maxAzs: 3, natGateways: 1});
+const stack = new cdk.Stack(app, 'integ-capacity-provider8', { env });
+
+// const vpc = new ec2.Vpc(stack, 'Vpc', { maxAzs: 3, natGateways: 1});
+const vpc = ec2.Vpc.fromLookup(stack, 'Vpc', { isDefault: true })
 
 const cluster = new ecs.Cluster(stack, 'Cluster', { vpc });
+Array.isArray(cluster)
 
 // create the 1st capacity provider with on-demand t3.large instances
-const cp = cluster.addCapacityProvider('CP', {
+cluster.addCapacityProvider('CP', {
```

Comment:
This is very neat! Thank you so much for making this adding cp experience better!